api-tests/wallet_payment_api.cards.tests.json

```diff
@@ -0,0 +1,180 @@
+{
+	"info": {
```

Comment:
Delete this duplicated collection, use the one from wallet-tests

Reply:
mea culpa, I see only now that the content of this collection differs from the one into wallet-tests.
btw what about move it into wallet-tests repo (changing name) and use here the one from that repo? this way we could re-use this collection for other repositories if needed without duplicating it